### PR TITLE
rel: 0.5.1

### PR DESCRIPTION
**Pull request by mgxd**
* updated changelog
* add util script to help future changelog generation
* bump version

@yarikoptic let's get everything in order to release for this coming Monday

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,20 +4,29 @@ All notable changes to this project will be documented (for humans) in this file
...
+- Helper to set metadata restrictions correctly
+- Usage is now shown when run without arguments
+- New fields to Seqinfo
+  - series_uid
+- Reproin heuristic support for xnat
 ### Changed
-
-### Deprecated
-
+- Dockerfile updated to use `dcm2niix v1.0.20180328`
```
Comment (yarikoptic) [inline CHANGELOG.md]: while at it -- let's test/update to current (uploaded yesterday to neurodebian) 1.0.20180622?

**Comment by mgxd** [comment]
@yarikoptic do you need anything else before tagging/pushing to pypi?

**Comment by yarikoptic** [comment]
Let's not wait for me... If anything, there will be .2 ;-)

**Comment by mgxd** [comment]
Looks like you are the sole maintainer on pypi 😄

**Comment by yarikoptic** [comment]
Later today then I guess I will upload and add you to the team. The same login mgxd?

**Comment by yarikoptic** [comment]
@mgxd I added you on pypi

**Comment by mgxd** [comment]
ok - in it goes

**Pull request merged**